pages/mentors/[id].js

```diff
@@ -52,26 +50,23 @@ export default function MentorPage() {
   }
 
   if (!data) {
-    return <>Looading</>;
+    return <>Loading</>;
   }
 
   return (
     <>
-      <Container mt="10" maxW={"5xl"} py={12}>
-        <SimpleGrid
-          columns={{ base: 1, md: 2 }}
-          minChildWidth={400}
-          spacing={10}
-        >
-          <Stack spacing={4}>
-            <Box height="5px" width="150px" bg="brand.orange" mb={2}></Box>
-            <Heading size="2xl" color="gray.700">
-              {data.name}
-            </Heading>
+      <Container maxW={"5xl"} py={12}>
+        <Stack spacing={4} marginRight={"auto"}>
+          <Heading color="gray.700">
+            {data.name}
+          </Heading>
+          {data.title &&
             <Heading size="md" color="gray.500">
               {data.title}
             </Heading>
-            <Stack direction={"row"} justifyContent={"left"}>
+          }
+          {data.twitter || data.linkedin &&
```

Comment:
somehow the condition must be enclosed in parenthesis otherwise it won't render (e.g. https://deploy-preview-126--tupu-app.netlify.app/mentors/rec_c8ei3pt17c43qguaejsg )

```suggestion
          {(data.twitter || data.linkedin) &&
```

---

<img width="291" alt="image" src="https://user-images.githubusercontent.com/6698585/229636628-048a730e-c443-451f-9763-b381f4fb2b3e.png">